src/redux/categories/categories.js

```diff
@@ -0,0 +1,20 @@
+const CHECK_STATUS = 'CHECK_STATUS';
+
+const checkStatusReducer = (state = [], action) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return state.filter((category) => category.id === action.id);
+
```

Comment:
- [ ] Your state is not returning a 'Check Status' string. Kindly make your case something like above ⬇️

```
 case CHECK_STATUS:
      return action.payload;
```